app/controllers/api/v1/invite_requests_controller.rb

```diff
@@ -0,0 +1,89 @@
+class Api::V1::InviteRequestsController < ApplicationController
+    def get_invites
+        id = request.headers['id'].to_i
+
+        invites = InviteRequest.where(sender_id: id).or(InviteRequest.where(receiver_id: id))
+
+        invites = invites.where(status: true)
+
+        user_array = []
+
+        invites.each do |invite|
+            if invite[:sender_id] == id
+                user = User.find_by(id: invite[:receiver_id])
+                user = {id: user[:id], firstName: user[:first_name], lastName: user[:last_name]}
+                user_array.push(user)
+            else
+                user = User.find_by(id: invite[:sender_id])
+                user = {id: user[:id], firstName: user[:first_name], lastName: user[:last_name]}
```

Comment:
You should use dot syntax here to access the user's attributes i.e. user.id, user.first_name, user.last_name